README.adoc

```diff
@@ -92,17 +92,31 @@ Ruby code is in `lib/`
 == Other useful things
 
  - There is a `CMakeLists.txt` that's mainly there for CLion support; we don't use cmake to build any of this.
- - You can use vagrant to bootstrap a VM to test under Linux while on Darwin; this is useful when testing `seccomp`.
+ - You can use UTM to bootstrap an x86_64 VM to test with Linux while on MacOS; this is useful when testing `seccomp`.
```

Comment:
```suggestion
 - You can use UTM to bootstrap an x86_64 VM to test with Linux while on MacOS; this is useful when testing `seccomp`.
  - You can get the ISO of the latest ESS compatible version of Ubuntu [here](https://releases.ubuntu.com/22.04/?_gl=1*1w2295z*_gcl_au*NjQ3Njk3OTAwLjE3MjAwMTM3MzA.&_ga=2.45506340.1872351921.1720124619-993441323.1720124619)
```

I think it'd be handy to have a link to the ISO, especially while the ubuntu 24 failures persist